cg2019_paper.tex

```diff
@@ -507,9 +508,9 @@ \section{Genetic Algorithm}
 
 With regard to the genetic operators, mutation has remained the same as in previous approaches of our genetic controller, i.e. \textbf{non uniform mutation} \cite{mutation1997}. 
 
-A new \textbf{Pole Position Selection policy} (or race-based selection) has been implemented in this approach, aiming to get better or more reliable individuals/controllers to be parents of the following population. To this end all the individuals are arranged in groups of 10, then some different races of  several laps are simulated using every individual as a controller (with the same car) in a track of TORCS. After every race, the participants obtain different scores depending on their position in the final rank. The best 5 controllers in the sum of all the races are selected as parents for the following offspring.
+A new \textbf{Pole Position Selection policy} (or race-based selection) has been implemented in this approach, aiming to get better or more reliable individuals/controllers to be parents of the following population. To this end all the individuals are arranged in groups of 10, then some different races of  several laps are simulated using every individual as a controller (with the same car) in a track of TORCS. After every race, the participants obtain different scores depending on their position in the final rank. The best 5 controllers in the \textcolor{red}{total} of all the races are selected as parents for the following offspring.
 
-This way, the best individuals will be selected to reproduce with higher probability. It is not possible to assure they are absolutely the best, due to the uncertainty present in this type of environments, i.e. games against non-deterministic opponents \cite{merelo2016statistical}. However, we argue that this selection policy will be `less sensitive' to that uncertainty (or noise), and thus, it will be fairer and more reliable than an approach purely based on the fitness values. Thus, we think that this proposed selection mechanism would benefit getting a good optimization process.
+This way, the best individuals will be selected to reproduce with higher probability. It is not possible to \textcolor{red}{guarantee} they are absolutely the best, due to the uncertainty present in this type of environments, i.e. games against non-deterministic opponents \cite{merelo2016statistical}. However, we argue that this selection policy will be `less sensitive' to that uncertainty (or noise), and thus, it will be fairer and more reliable than an approach purely based on the fitness values. Thus, we think that this proposed selection mechanism would \textcolor{red}{create} a good optimization process.
 %Due to the high-demanding time this method is, this race-based selection process is just conducted every 5 generations (not in all of them).
```

Comment:
Don't like benefit getting, but create is not the word either. Be a part? Compose? Conform?

Reply:
I read the sentence as “a good optimization process would be the result of the proposed selection mechanism”, that is, that it would be a direct route from one to the other. If “a good optimization process” is made up of several parts, then “be a part” would be correct. Another options would be something like “produce”, “deliver” or even “return”.

Reply:
"deliver" is a good one.

Reply:
Sorry but this is not the right paper to revise.
the right one is  'TOG2019-R2.tex' in the folder Tog